rest_framework/routers.py

```diff
@@ -301,10 +301,7 @@ class APIRootView(views.APIView):
     def get(self, request, *args, **kwargs):
         # Return a plain {"name": "hyperlink"} response.
         ret = OrderedDict()
-        namespace = request.resolver_match.namespace
         for key, url_name in self.api_root_dict.items():
-            if namespace:
-                url_name = namespace + ':' + url_name
```

Comment:
Note: these lines are made redundant by the changes to `reverse()`. The tests below in `TestRootView` verify this behavior.